docs/sdks/server-sdks/node/quickstart.mdx

```diff
@@ -0,0 +1,313 @@
+The SDK provides different assignment functions based on the type of value you need:
+
+| Function | Return Type |
+|----------|-------------|
+| `getStringAssignment()` | String |
+| `getBooleanAssignment()` | Boolean |
+| `getJSONAssignment()` | JSON object |
+| `getIntegerAssignment()` | Integer |
+| `getNumericAssignment()` | Float |
+
+:::note
+See more details about assignment functions in the [Assignments](/sdks/client-sdks/javascript/assignments) page.
+:::
+
+### Using Assignments
+
+After receiving an assignment, your application should implement logic to modify the user experience accordingly:
+
+```javascript
+// Example using a boolean assignment
+const variation = eppoClient.getStringAssignment(
+  featureKey,
+  subjectKey,
+  subjectAttributes,
+  defaultValue
+);
+
+// Render different components based on assignment
+if (variation === "landing-page-a") {
+    return <LandingPageA />
+} else if (variation === "landing-page-b") {
+    return <LandingPageB />
+} else {
+    return <LandingPageC />
+}
+```
+
+That's all the code you need to make a basic assignment. After that, you can do the rest of the work running the experiment from the Eppo UI. This is awesome both because it allows you to iterate quickly and because it gives you the ability to change the experiment configuration without redeploying your code.
+
+You can see the full steps for setting up a feature flag including UI steps in the [feature flag quickstart](/feature-flag-quickstart).
+
+
+## Experiments
+
+While feature flags are useful, they do not send you any information about how your users are interacting with the feature. Experiments provide a way to collect data about these interactions using whichever logging and data warehousing system you prefer.
+
+To log events through the SDK, you need to define a `logAssignment()` function using the `IAssignmentLogger` interface to pass to the `init()` function.
+
+For simplicity, we'll create a `logAssignment()` function that logs the assignment data to the console.
+
+```javascript
+// Import Eppo's assignment logger interface and client initializer
+import { IAssignmentLogger, init } from "@eppo/js-server-sdk";
+
+// Define logAssignment so that it logs events
+const assignmentLogger: IAssignmentLogger = {
+  logAssignment(assignment) {
+    console.log(assignment);
+  }
+};
+
+// Initialize the SDK
+await init({
+  apiKey: "<SDK_KEY>",
+  assignmentLogger,
+});
+
+```
+
+
+:::note
+In a production application, you would want to replace the `console.log()` statement with an actual logging system. We have documentation on how to set up logging with multiple popular data warehouses and logging systems in the [Assignment page](/sdks/client-sdks/javascript/assignments/#logging-data-to-your-data-warehouse).
+:::
+
+### Assign a variant
+
+After initializing the SDK with the `logAssignment()` function, you can make assignments in the same way as for feature flags.
+
+```js
+import * as EppoSdk from "@eppo/js-server-sdk";
```

Comment:
Same as before, this import is the wrong package name, inconsistent quotes, and we should use a back-end example